### DISCUSSION TITLE
regarding documentation

### DISCUSSION THREAD

**Discussion by Rahuifcnja4** [badeball/cypress-cucumber-preprocessor#880] [General]
plz provide documentation or video links how to use cypress 11 cucumber framework for js

**Comment by badeball** [comment]
https://github.com/badeball/cypress-cucumber-preprocessor/tree/master/examples

**Reply by Rahuifcnja4** [reply]
tq when we create esbuild bundler i came across one issue node file system not abel to use in our project

**Comment by harinyadav1985** [comment]
Hi, I need your help in understanding how I can use cucumber with Cypress 12.
